extensions/EXT/EXT_texture_format_BGRA8888.txt

```diff
@@ -88,6 +89,11 @@ New Tokens
 
         GL_BGRA_EXT                                     0x80E1
 
+    Accepted by the <internalformat> parameter of TexImage2D, unless using
+    OpenGL ES versions prior to 3.0:
```

Comment:
The more I think about it, the more I think that limiting this to ES 3 is a mistake. I think when I proposed this, I thought the sized internal format support for ES 2 was more limited than it is. But even if it is, it means we need to complicate implementations and the CTS tests for little reason here.

Thoughts? Should this just apply to all versions instead?

Reply:
Supporting all versions has been approved by the WG.

Reply:
OK. Added a commit to include ES2. I don't think changing ES1 is a good idea, sorry for not being clear about that earlier.

Reply:
I also decided to keep the interaction with the ES3 spec, even if it could be removed now. It seems clearer this way.

Reply:
Looking at [the minutes](https://gitlab.khronos.org/opengl/API/-/wikis/OpenGL%7CES-Meetings/2024-05-22-minutes), it seems like adding ES2 is what was approved, so I assume this makes the approval stand?

Reply:
That's correct. I meant that supporting ES2 was approved in my previous statement, not that ES1 is supported.